src/SampleLibraryZeroTouch/SampleLibraryZeroTouch.csproj

```diff
@@ -24,7 +24,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DocumentationFile>bin\Debug\$(UICulture)\SampleLibraryZeroTouch.XML</DocumentationFile>

```

Comment:
any idea why this was using a UICulture?

Reply:
No.

Reply:
It is used in Dynamo.